src/inmanta/moduletool.py

```diff
@@ -340,11 +340,16 @@ def modules_parser_config(cls, parser: ArgumentParser):
         build = subparser.add_parser("build", help="Build a Python package from a V2 module.")
         build.add_argument(
             "path",
-            help="The path to the module that should be built",
+            help="The path to the module that should be built. By default, the current working directory is used.",
             nargs="?",
         )
         build.add_argument(
-            "-o", "--output-dir", help="The directory where the Python package will be stored.", default=None, dest="output_dir"
+            "-o",
+            "--output-dir",
+            help="The directory where the Python package will be stored. By default, the ./dist directory "
+            "in the root of the module is used.",
```

Comment:
This seems to contradict itself. I'd interpret `./dist` to mean the `$(pwd)/dist`, while "in the root of the module" implies `<module_root>/dist`. I think either the "./" or the "in the root of the module" should be dropped to describe either behavior correctly.